ops/deploy-ccs.groovy

```diff
@@ -51,21 +51,21 @@ def findAmis() {
 		platinumAmiId: sh(
 			returnStdout: true,
 			script: "aws ec2 describe-images --owners self --filters \
-			'Name=name,Values=bfd-platinum-??????????????' \
+			'Name=name,Values=bfd-amzn2-platinum-??????????????' \
 			'Name=state,Values=available' --region us-east-1 --output json | \
 			jq -r '.Images | sort_by(.CreationDate) | last(.[]).ImageId'"
     ).trim(),
     bfdPipelineAmiId: sh(
       returnStdout: true,
       script: "aws ec2 describe-images --owners self --filters \
-			'Name=name,Values=bfd-etl-??????????????' \
+			'Name=name,Values=bfd-amzn2-etl-??????????????' \
 			'Name=state,Values=available' --region us-east-1 --output json | \
 			jq -r '.Images | sort_by(.CreationDate) | last(.[]).ImageId'"
     ).trim(),
     bfdServerAmiId: sh(
       returnStdout: true,
       script: "aws ec2 describe-images --owners self --filters \
-			'Name=name,Values=bfd-fhir-??????????????' \
+			'Name=name,Values=bfd-amzn2-fhir-??????????????' \
```

Comment:
Namespace AL2 AMIs for auditing and easy rollback if required.

Reply:
I verified there are recently updated gold AL2 images:
```
"2021-08-31T22:23:40.000Z"
"2021-09-01T12:53:08.000Z"
"2021-09-01T13:59:46.000Z"
"2021-09-07T16:38:07.000Z"
```